sass/digital-exchange/components/SearchBar.scss

```diff
@@ -0,0 +1,31 @@
+.SearchBar {
+  &__container {
+    display: flex;
+  }
+
+  &__textbox {
+    display: flex;
+    flex: 1;
+    border: 1px solid $color-de-list-icon;
+    border-right: 0;
+  }
+
+  &__textbox--base {
+    flex: 1;
+    padding: 0 5px;
+    border: 0;
+    outline: none;
+  }
+
+  &__button {
+    padding-right: 12px;
+    padding-left: 12px;
+    border-color: $color-de-list-icon;
+    box-shadow: none;
+  }
+
+  &__button-close {
+    padding-right: 12px;
+    padding-left: 12px;
+  }
+}
```

Comment:
please add one empty line at the end of the file

Reply:
noted @gidesan

Reply:
done @gidesan